### ISSUE TITLE
:wheelchair: a11y(Bal-nav): Open A11y Issues

### ISSUE THREAD

**Issue by GinaBiondo** [baloise/design-system#1621]
### Description of this issue

The following accessibility issues are still open.

1. Meta Nav
     - the nav has a role="tablist" hence the nav is not announced as navigation
     - links should be in a list (ul > li)
     - there should be aria-labels on the nav tags to distinguish between "meta", "main" and "sub" navigations

2. Main nav
     - when the flyout is opened, the first element inside should be in focus

3. Meta Nav Popups
     - users are currently stuck in the popup , when navigating via Screenreader (Keyboard) to a button, link, input field. 

### How to fix it

See above

### Code Reproduction URL

https://www.baloise.ch/de/privatkunden.html

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by mmarinkov** [comment]
1. Meta Nav
     - the nav has a role="tablist" hence the nav is not announced as navigation = will check
     - links should be in a list (ul > li) = will check
     - there should be aria-labels on the nav tags to distinguish between "meta", "main" and "sub" navigations
       - i guess the labels are not understandable by the users, moreover we need all the tranlstions for them

2. Main nav
     - when the flyout is opened, the first element inside should be in focus
       - all tabs are workling the same way, we should not change the behaviour of one case  

3. Meta Nav Popups
     - users are currently stuck in the popup , when navigating via Screenreader (Keyboard) to a button, link, input field. 
       - user can navigate with the keyboard to the close element on the top right or even just click escape

**Comment by carolabes** [comment]
Hi @mmarinkov,

1. Meta Nav
  we can use translation keys for the aria-label, but first we need to know, if those aria-labels already exists or need to be added by you.

2. main nav
   Since the main nav is a core feature of the page, it requires a high level of accessibility. So the focus order needs to be correct and mouse operation should equal key operation. Therefore we should think of adding this case or find an alternative to provide this functionality.

3. Meta nav popup
   This is a voiceOver Screenreader issue only. I tested this on macOS, voiceOver, Chrome. The user can't navigate back to the close button once the focus is on a link or button. This is especially an issue for the search popup since there is a tab componenten below the button which the screenreader user can never reach. It would be great if you could find a fix for this.

**Comment by hirsch88** [comment]
3. is a standard safari behaviour. when enabling the options it works

![Image](https://github.com/user-attachments/assets/fe0a99d7-c6ca-495f-aadb-ee4648b74638)

**Comment by mmarinkov** [comment]
Summary ticket
1. Has been implemented the only part missing is:

- [ ] the right naming conventions and translations for the meta/mai nav from IBM

2. Focus in Flyout

- [ ] we should follow this example:https://www.ibm.com/de-de. In our nav we cannot navigate inside the flayout 

3. Is a standard safari behaviour. when enabling the options it works

**Comment by hirsch88** [comment]
![Image](https://github.com/user-attachments/assets/2b634bdb-e6ab-4ecc-a8ba-0e2915cc7fd0)

1 Mainnavigation
2 Navigation
3 Subnavigation (falls benötigt in diesem Bereich
4 Inpage Navigation (nächster Screenshot)

**Comment by hirsch88** [comment]
created #1639  and #1638
